source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,96 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+If you want to work primarily with JSON strings in your application, 
+use the ``JsonObject`` class. For applications that work with JSON
+strings, ``JsonObject`` is a more performant alternative to both ``Document``
+and ``BsonDocument`` as ``JsonObject`` does not create an intermediate
+``Map`` object.
```

Comment:
```suggestion
The ``JsonObject`` class acts as a wrapper for JSON strings.
``JsonObject`` is a more performant alternative to both ``Document``
and ``BsonDocument`` when working with JSON strings because it does not create an intermediate
``Map`` object.
```

Reply:
Thanks for the note Nathan. I agree that your suggested change is an improvement.

Takeaways:

Be careful of unnecessarily reusing words/rephrasing. Try to express linear thoughts.